src/sonic-ctrmgrd/tests/container_test.py

```diff
@@ -0,0 +1,535 @@
+import unittest
+from unittest.mock import MagicMock, patch
+import imp
```

Comment:
In Python 3, `imp` is deprecated, superseded by `importlib`. Prefer `importlib`. Also please reorganize as per other similar comments.

Reply:
done